src/wp-includes/block-editor.php

```diff
@@ -648,6 +648,23 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 		$editor_settings['postContentAttributes'] = $post_content_block_attributes;
 	}
 
+	// Expose block bindings sources in the editor settings.
+	$registered_block_bindings_sources = get_all_registered_block_bindings_sources();
+	if ( ! empty( $registered_block_bindings_sources ) ) {
+		// Initialize array.
+		$editor_settings['blockBindings'] = array();
```

Comment:
Should the setting be called `blockBindings` or `blockBindingsSources`? Any thoughts?

Reply:
It's a source, so should have the source word.

Reply:
👍 Changed in this commit: https://github.com/WordPress/wordpress-develop/pull/7020/commits/876155ffda8835b1ce24ce98b13ff1854c08c34c